Samples/Swift/nhubsample/nhubsample/ViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ViewController.swift
+//  nhubsample
+//
+//  Created by TOYS on 2/25/19.
```

Comment:
All files should have the standard header:
```
//
//  Copyright © 2018 Microsoft All rights reserved.
//  Licensed under the Apache License (2.0).
//
```

Please remove replace these headers in all new files.